src/libraries/System.Private.CoreLib/src/Internal/Padding.cs

```diff
@@ -18,7 +18,7 @@ internal static class PaddingHelpers
 
     /// <summary>Padding structure used to minimize false sharing</summary>
     [StructLayout(LayoutKind.Explicit, Size = PaddingHelpers.CACHE_LINE_SIZE - sizeof(int))]
-    internal struct PaddingFor32
+    internal readonly struct PaddingFor32
```

Comment:
This one has a duplicated definition here where CACHE_LINE_SIZE may be different in ARM64. 
https://github.com/dotnet/runtime/blob/38b6c82216c157a5631a433c23ae3c60d0d7f647/src/libraries/Common/src/Internal/Padding.cs#L17

Reply:
https://github.com/dotnet/runtime/issues/80650#issuecomment-1382951042